.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: false
+language: perl
+matrix:
+    fast_finish: true
+perl:
```

Comment:
Can probably remove this if we're moving to CircleCI

Reply:
Sorry I forgot to push my commit :)